ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -1248,30 +1249,30 @@ objects:
         }
 
         function verify_teardown_playbooks() {
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-load-balancers.yaml" --list-tasks
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-compute-nodes.yaml" --list-tasks
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-control-plane.yaml" --list-tasks
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-bootstrap.yaml" --list-tasks
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-network.yaml" --list-tasks
-          ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-security-groups.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-load-balancers.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-compute-nodes.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-control-plane.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-bootstrap.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-network.yaml" --list-tasks
+          ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-security-groups.yaml" --list-tasks
 
           # down-containers.yaml was introduced in 4.6
-          if [ -f "${OS_UPI_DIR}/down-containers.yaml" ]; then
-                ansible-playbook -vvv -i "${OS_UPI_DIR}/inventory.yaml" "${OS_UPI_DIR}/down-containers.yaml" --list-tasks
+          if [ -f "${ASSETS_DIR}/down-containers.yaml" ]; then
+                ansible-playbook -vvv -i "${ASSETS_DIR}/inventory.yaml" "${ASSETS_DIR}/down-containers.yaml" --list-tasks
```

Comment:
Might want to move `--list-tasks` to the front as you did above.

Reply:
I think I do that in the next commit (you might be looking at the diff for a single commit). Here's the final product: https://github.com/openshift/release/pull/10944/files#diff-3cfd01931f3f21aec94d18d4d0b42a20R1261